<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDebug</class>
 <widget class="QWidget" name="ConfigureDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>Global Log Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="log_filter_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="toggle_console">
          <property name="text">
           <string>Show Log in Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_log_button">
          <property name="text">
           <string>Open Log Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="extended_logging">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>When checked, the max size of the log increases from 100 MB to 1 GB</string>
        </property>
        <property name="text">
         <string>Enable Extended Logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>This will be reset automatically when yuzu closes.</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Homebrew</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Arguments String</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="homebrew_args_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Graphics</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="enable_graphics_debugging">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>When checked, the graphics API enters a slower debugging mode</string>
        </property>
        <property name="text">
         <string>Enable Graphics Debugging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enable_nsight_aftermath">
        <property name="toolTip">
         <string>When checked, it enables Nsight Aftermath crash dumps</string>
        </property>
        <property name="text">
         <string>Enable Nsight Aftermath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="disable_macro_jit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>When checked, it disables the macro Just In Time compiler. Enabling this makes games run slower</string>
        </property>
        <property name="text">
         <string>Disable Macro JIT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="disable_loop_safety_checks">
        <property name="toolTip">
         <string>When checked, it executes shaders without loop logic changes</string>
        </property>
        <property name="text">
         <string>Disable Loop safety checks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Debugging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QCheckBox" name="fs_access_log">
        <property name="text">
         <string>Enable FS Access Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reporting_services">
        <property name="text">
         <string>Enable Verbose Reporting Services</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>This will be reset automatically when yuzu closes.</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QCheckBox" name="quest_flag">
        <property name="text">
         <string>Kiosk (Quest) Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enable_cpu_debugging">
        <property name="text">
         <string>Enable CPU Debugging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_debug_asserts">
        <property name="text">
         <string>Enable Debug Asserts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_auto_stub">
        <property name="text">
         <string>Enable Auto-Stub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>This will be reset automatically when yuzu closes.</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>log_filter_edit</tabstop>
  <tabstop>toggle_console</tabstop>
  <tabstop>extended_logging</tabstop>
  <tabstop>open_log_button</tabstop>
  <tabstop>homebrew_args_edit</tabstop>
  <tabstop>enable_graphics_debugging</tabstop>
  <tabstop>enable_nsight_aftermath</tabstop>
  <tabstop>disable_macro_jit</tabstop>
  <tabstop>disable_loop_safety_checks</tabstop>
  <tabstop>reporting_services</tabstop>
  <tabstop>quest_flag</tabstop>
  <tabstop>use_debug_asserts</tabstop>
  <tabstop>use_auto_stub</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
